node/configuration.yaml

```diff
@@ -305,6 +307,8 @@ mainnet_base: &mainnet_base
     blockRetrievalQueueSize: 100
     propagationQueueSize: 100
     pendingTxResubmissionPeriod: 7 # seconds
+    walletProductionApi: true
```

Comment:
Interestingly, there are some differences (which I cannot understand why GH is not showing). Unless [diffChecker](https://www.diffchecker.com/3gaPNqt6) lies here, the block was bumped. The relevant excerpts were taken from:

- https://github.com/input-output-hk/cardano-sl/blob/master-release-candidate/node/configuration.yaml#L204-L311
- https://github.com/input-output-hk/cardano-sl/blob/cardano-sl-1.0-exchanges/node/configuration.yaml#L204-L311

@gromakovsky , @dcoutts , am I being mental? Better being safe then sorry 😉

Reply:
This is correct version (in master):
```
  update: &mainnet_base_update
    applicationName: cardano-sl
    applicationVersion: 0
    lastKnownBlockVersion:
      bvMajor: 0
      bvMinor: 1
      bvAlt: 0
    systemTag: none
```

The fact that it's wrong in exchanges branch doesn't matter, because exchanges don't create blocks and don't use update system for software updates.

Reply:
Perfect @gromakovsky , that's all I wanted to hear 😉